packages/snaps-sdk/CHANGELOG.md

```diff
@@ -6,6 +6,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.0.0]
+### Added
+- **BREAKING:** Add non-restricted RPC method `snap_resolveInterface`, which allows a snap to resolve a given user interface bound to a `snap_dialog` with a custom value. ([#2509](https://github.com/metamask/snaps/pull/2509))
```

Comment:
While this PR is breaking, it doesn't break `snaps-sdk`. We may want to do a minor bump here.

```suggestion
- Add non-restricted RPC method `snap_resolveInterface` ([#2509](https://github.com/metamask/snaps/pull/2509))
   - This method allows a Snap to resolve a given user interface bound to a `snap_dialog` with a custom value.
```

Reply:
Downgraded `snaps-sdk` to minor release: https://github.com/MetaMask/snaps/pull/2581/commits/79e11a92113b1e4860cbe0bf73cb75362a7ff08e